src/Resources/config/routing.yml

```diff
@@ -1,33 +1,30 @@
 simple_things_entity_audit_home:
     path: /{page}
+    controller: SimpleThings\EntityAudit\Controller\AuditController::indexAction
```

Comment:
Are you sure the controllers are defined like services here?

Reply:
It is instance of `ContainerAwareInterface` so it should be trait like FQCN and autogenerate like with `SimpleThingsEntityAuditBundle:Audit:index`. I will check it.

Reply:
@jordisala1991 
After check it is working.